File: back-end/app/Models/HorarioDisponible.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Attributes\Fillable;
use Illuminate\Database\Eloquent\Model;

#[Fillable([
    'dia_semana',
    'hora_inicio',
    'hora_fin',
    'activo',
])]
class HorarioDisponible extends Model
{
    protected $table = 'horarios_disponibles';

    public $timestamps = false;

    protected function casts(): array
    {
        return [
            'activo' => 'boolean',
        ];
    }
}

File: wearable/app/src/main/java/com/nailsstudio/wear/data/model/HorarioDisponible.kt
package com.nailsstudio.wear.data.model

import com.google.gson.annotations.SerializedName

data class HorarioDisponible(
    val id: Long,
    @SerializedName("dia_semana") val diaSemana: String,
    @SerializedName("hora_inicio") val horaInicio: String,
    @SerializedName("hora_fin") val horaFin: String,
    val activo: Boolean,
)

data class HorariosDisponiblesResponse(
    @SerializedName("horarios_disponibles") val horariosDisponibles: List<HorarioDisponible>,
)
